<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentpage</class>
 <widget class="QMainWindow" name="studentpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="StudentSelect">
     <widget class="QPushButton" name="campDB_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DATABASE DISPLAY</string>
      </property>
     </widget>
     <widget class="QLabel" name="welcome_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome, Student</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="campDB_texEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>See Campus Database:</string>
      </property>
     </widget>
     <widget class="QLabel" name="tourType_label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Tour Type:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UCI_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>UCI</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Saddle_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>170</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SADDLEBACK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ASU_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>280</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ASU</string>
      </property>
     </widget>
     <widget class="QPushButton" name="custom_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>380</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CUSTOM</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="UCI_textEdit">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>70</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="plainText">
       <string>Start at the University of California, Irvine and tour 11 college campuses.</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="Saddle_textEdit">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>170</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="plainText">
       <string>Start at Saddleback Community College, tour 13 college campuses (including 2 new campuses).</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="ASU_textEdit">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>280</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="plainText">
       <string>Start at Arizona State University, choose a # of other other colleges to visit.</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="custom_textEdit">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>380</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="plainText">
       <string>Choose the colleges that you wish to tour.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_button_select">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tour_textEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>291</width>
        <height>271</height>
       </rect>
      </property>
      <property name="plainText">
       <string>In this program, You will be able to select one of these 4 tour type in order create your desired college tour plan. After your colleges have been selected, you will be able to purchase souvenirs from the campuses you visit. Lastly, you will get a sorted college tour plan and a souvenir receipt from your trip.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="databaseDisplay">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>781</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="campus_combo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>281</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="campus_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Campus</string>
      </property>
     </widget>
     <widget class="QLabel" name="database_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Database</string>
      </property>
     </widget>
     <widget class="QPushButton" name="other_button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>450</y>
        <width>141</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Other Campuses</string>
      </property>
     </widget>
     <widget class="QPushButton" name="souv_button">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>450</y>
        <width>141</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Souvenirs</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_button_db">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>10</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="startCollegeTour">
     <widget class="QLabel" name="start_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>390</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Starting Campus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enter_button_1">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>430</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_button_custom">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>20</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="avali_label_1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Avaliable Campuses</string>
      </property>
     </widget>
     <widget class="QComboBox" name="start_combo">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>390</y>
        <width>231</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="start_listView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>361</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="collegeTour">
     <widget class="QLabel" name="next_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Next Campus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="done_button">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>430</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enter_button_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>430</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QListView" name="select_listView">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>60</y>
        <width>361</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="avali_listView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>361</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="avali_label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Avaliable Campuses</string>
      </property>
     </widget>
     <widget class="QLabel" name="select_label">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Selected Campuses</string>
      </property>
     </widget>
     <widget class="QComboBox" name="next_combo">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>390</y>
        <width>251</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="back_button_general">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="souvenirShop">
     <widget class="QComboBox" name="selectCampus_comboBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="souv_tableView">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>271</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="souvCart_tableView">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>90</y>
        <width>371</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="backFromSouv_button">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Back </string>
      </property>
     </widget>
     <widget class="QPushButton" name="addSouvenir_button">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QComboBox" name="souv_comboBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="quantity_spinBox">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>360</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="souv_label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Souvenirs of: </string>
      </property>
     </widget>
     <widget class="QLabel" name="souvCart_label">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Souvenir Cart</string>
      </property>
     </widget>
     <widget class="QLabel" name="total_label">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>370</y>
        <width>56</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="dollarSign">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>360</y>
        <width>31</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>$</string>
      </property>
     </widget>
     <widget class="QPushButton" name="checkout_button">
      <property name="geometry">
       <rect>
        <x>402</x>
        <y>450</y>
        <width>371</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Checkout</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalCart_label">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>370</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="asuCollegeTour">
     <widget class="QPushButton" name="back_button_asu">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enter_button_asu">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QLabel" name="asu_label_title">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>421</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ASU College Tour - Starting at ASU</string>
      </property>
     </widget>
     <widget class="QLabel" name="asu_label_prompt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>501</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter the number of colleges you would like to visit (including ASU):</string>
      </property>
     </widget>
     <widget class="QComboBox" name="asu_combo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SortCollegeTour">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>331</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="total_dist_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Total Distance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="distanceNum_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>miles</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>449</x>
        <y>80</y>
        <width>321</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="souvTitleLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Souvenirs</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QListView" name="souvFinal_listView"/>
         </item>
         <item>
          <widget class="QListView" name="quantityFinal_listView"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QPushButton" name="finish_button">
           <property name="text">
            <string>Finish</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="totalCost_label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Total Cost:  $</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalNum_label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="startCampus_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>82</y>
        <width>331</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Sorted College Tour Plan</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QListWidget" name="sort_listWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
